.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -38,6 +44,6 @@ jobs:
 
       - name: Creation of issue in other Repositories
         env:
-          GITHUB_TOKEN: ${{ github.token }}
+          GITHUB_TOKEN: ${{ secrets.ISSUES_WRITE_TOKEN }}
```

Comment:
Since we are dealing with multiple repositories `github.token` will not be enough.

You need to create a personal access token.

I followed these steps and called it `ISSUES_WRITE_TOKEN`:
1. Click on profile picture on top right of the screen
2. Click `Settings`
3. On the left menu, go all the way down to `<> Developer Settings`
4. Click `Personal access tokens`
5. Click `Fine-grained tokens`
6. Click `Generate new token`
7. Choose name, description, repositories, etc..
![image](https://user-images.githubusercontent.com/23459549/215042739-0e576f5a-b3b4-425c-89db-b7abc889e37c.png)
8. Click `Repository permissions` and find `Issues`. Choose `Read and Write` access
![image](https://user-images.githubusercontent.com/23459549/215042845-88faea04-f6ab-4500-97a8-ba9133cbc8f3.png)
9. Generate token and copy
10. Go to `checkstyle` repository and click `Settings`
11. On the left menu, under `Security`, click `Secrets and variables` and then `Actions`
12. Click `New repository secret` and add it with the name of `ISSUES_WRITE_TOKEN`
![image](https://user-images.githubusercontent.com/23459549/215043389-49ad27cc-9836-4ad4-ac65-4465ba45f542.png)
13. Add token and you should be good to go

Reply:
there is problem with this, as I donot see repositories of checkstyle organization, I see only personal repos

![image](https://user-images.githubusercontent.com/812984/215774107-ac2c3a56-ef96-436e-b861-23f2ff564bb1.png)

Reply:
Lets move this activities in other repositories to other workflow, or simply use PAT  (that already exists in our secrets)

Reply:
Will keep this in mind in the future.